bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java

```diff
@@ -248,12 +252,32 @@ private String getChecksList(PullRequestCheckIssueVisitor visitor) {
         }
     }
 
-    private String getStatusMessage(List<Review> reviews, PullRequestCheckIssueVisitor visitor) throws IOException {
+    private String getStatusMessage(List<Review> reviews, PullRequestCheckIssueVisitor visitor) {
         var progressBody = new StringBuilder();
         progressBody.append("Progress\n");
         progressBody.append("--------\n");
-
         progressBody.append(getChecksList(visitor));
+
+        var issue = Issue.fromString(pr.getTitle());
+        if (issueProject != null && issue.isPresent()) {
+            progressBody.append("\n\nIssue\n");
+            progressBody.append("--------\n");
```

Comment:
Might be easier to use `##` instead of `------`?

Reply:
Sure, can change.